lib/src/utils.dart

```diff
@@ -2,25 +2,51 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-final _identifier = RegExp(r'^([(_|$)a-zA-Z]+([_a-zA-Z0-9])*)$');
```

Comment:
The `(_|$)` is very likely a mistake since it allows identifiers to start with `(`, `|` or `)`.

Reply:
Oh wow.  Good catch!